public/static/locales/en/oauth.json

```diff
@@ -0,0 +1,15 @@
+{
+    "invalid_request": "The authorization request sent to the remote system by the DE was invalid. Please contact support.",
+    "unauthorized_client": "The DE is not authorized to request access to the remote system. Please contact support.",
```

Comment:
Just picking a random spot - I agree we should have 411 review the text.  I wouldn't be surprised if a user thought "remote system" meant their system and they started messing with permission settings on their computer or something like that 😄